app/facades/usda_facade.rb

```diff
@@ -0,0 +1,13 @@
+class UsdaFacade
+
+  def self.name_search_items(name)
+    UsdaService.name_search_items(name).map do |food|
+      Food.new(food)
+    end
+  end
+
```

Comment:
Same thing here. If you add additional params (see note in your service) I think you could have a single method that gives you all that you want.